packages/opencensus-instrumentation-grpc/src/grpc.ts

```diff
@@ -223,6 +230,14 @@ export class GrpcPlugin extends BasePlugin {
             GrpcPlugin.ATTRIBUTE_GRPC_STATUS_CODE,
             grpcTypes.status.OK.toString());
       }
+
+      // record stats
+      const tags = new TagMap();
+      tags.set(serverMetrics.GRPC_SERVER_METHOD, {value: rootSpan.name});
```

Comment:
As an aside: I think this use case is a good reason to not add `Sent.` or `Recv.` to the gRPC span names until they are ready to be exported.

Reply:
hmm, makes sense. Looks like need to revise the Java's implementation.